browser/src/core/Socket.js

```diff
@@ -797,21 +797,22 @@ app.definitions.Socket = L.Class.extend({
 				}, 3000);
 			}
 
-			// Close any open dialogs first.
-			vex.closeAll();
+			if (textMsg === 'idle' || textMsg === 'oom') {
```

Comment:
I don't understand why we need this change, and how it's related to the document conflict case.

Reply:
It causes to close open vex window(s) even when a vex window will not open. I modified it while testing the "close: documentconflict" case.